contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -18,10 +19,9 @@ use cosmwasm_std::{
     MessageInfo, Response, StdError, StdResult, Uint128, Uint256, WasmMsg,
 };
 use cw2::set_contract_version;
+use cw20::Denom::Cw20;
```

Comment:
probably just leftover from the editor, but no need to import a specific variant of an enum
```suggestion
```